examples/examples/z_pong.rs

```diff
@@ -15,6 +15,7 @@ use clap::Parser;
 use zenoh::config::Config;
 use zenoh::prelude::sync::*;
 use zenoh::publication::CongestionControl;
+use zenoh::sample_builder::QoSBuilderTrait;
```

Comment:
Similar. Trait should be part of the prelude.

Reply:
doen